src/App.jsx

```diff
@@ -3,26 +3,58 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { request } from './api/api';
 
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
   };
 
+  componentDidMount() {
+    this.getTodos();
+  }
+
+  getTodos = async() => {
+    const todoListFromServer = await request('/todos');
```

Comment:
```suggestion
    const todoListFromServer = await getTodoss();
```
может '/todos'  перенести в api.js.